includes/Assets.php

```
@@ -692,9 +700,91 @@ public function get_scripts() {
     }
 
+    public function get_form_fields() {
+		$product_id = isset( $_GET['product_id'] ) ? intval( $_GET['product_id'] ) : 0; // phpcs:ignore
+		$product    = wc_get_product( $product_id );
+
+		$sections_data = [];
+
+
+        $sections = ProductFormFactory::get_sections();
+
+        foreach ( $sections as $key => $section ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)